### ISSUE TITLE
Authentication issue: redirect_uri_mismatch

### ISSUE THREAD

**Issue by krishh-konar** [kuzmoyev/google-calendar-simple-api#147]
Hi, 
I followed all the instructions for creating the credentials as mentioned in the documentation.
However, I am running into an issue with OAuth.

My code:
```
gc = GoogleCalendar(credentials_path=CONFIG["CALENDAR_API_CREDENTIAL_PATH"],
        token_path=CONFIG["CALENDAR_API_TOKEN_PATH"])

for calendar in gc.get_calendar_list():
    print(calendar)
```

Running this code leads me to browser to proceed with authorization with following output:
```
Please visit this URL to authorize this application: https://accounts.google.com/o/oauth2/auth?response_type=code&client_id=<some_id>&redirect_uri=http%3A%2F%2Flocalhost%3A8080%2F&scope=https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcalendar&state=<some_state>&access_type=offline
```

I notice that the `redirect_uri` here is set to `localhost:8080`, which eventually leads to OAuth failure.

<img width="322" alt="Screenshot 2022-10-21 at 2 26 33 PM" src="https://user-images.githubusercontent.com/11133427/197159587-67076e99-562b-4839-b3b3-aeefa8b3d68b.png">

The `redirect_uri` is already set in my `credentials.json`, but i doesnt seem to be picking it from there. Can you help me with what I'm doing wrong?

credential.json:

```
{
  "web": {
    "client_id": "<id>.apps.googleusercontent.com",
    "project_id": "notifier-app",
    "auth_uri": "https://accounts.google.com/o/oauth2/auth",
    "token_uri": "https://oauth2.googleapis.com/token",
    "auth_provider_x509_cert_url": "https://www.googleapis.com/oauth2/v1/certs",
    "client_secret": "<secret>",
    "redirect_uris": [
      "https://devopsdiary.tech"
    ],
    "javascript_origins": [
      "https://devopsdiary.tech"
    ]
  }
}
```

**Comment by krishh-konar** [comment]
Nevermind, it was a flaw in my understanding.
Apologies.

**Comment by kuzmoyev** [comment]
Hi, @krishh-konar.
Thank you for the submission, glad it's resolved now. Could you please share the solution, for future references?

**Comment by krishh-konar** [comment]
Well, there wasn't any solution as such. I made a typo while creating the credential while adding the domains in `redirect_uri`, so I updated it later on. After that it wasn't working. Then I left it and went to sleep and tried the next day, it was working. 

Later I just noticed there's a note attached at the bottom saying
```
Note: It may take 5 minutes to a few hours for settings to take effect
```

I guess it really did take a few hours xD
